### PR TITLE
[DO NOT MERGE - WIP][Doc] Add doc for using the Avro JSON schema definition in Python client

### PR DESCRIPTION
**Pull request by gaoran10**
### Motivation

Currently, there is a lack of how to use the Avro JSON schema definition in the Python client.

### Modifications

Add doc for using custom Avro JSON schema definition in Python client.

### CODE REVIEW

**Review by BewareMyPower — APPROVED** [review]

File: site2/docs/client-libraries-python.md [hunk 1]
```diff
@@ -224,6 +224,71 @@ while True:
...
+
+Users could load schema definition from file by `avro.schema` or `fastavro.schema`
+> refer to [load_schema](https://fastavro.readthedocs.io/en/latest/schema.html#fastavro._schema_py.load_schema) or [Avro Schema](http://avro.apache.org/docs/current/gettingstartedpython.html)
+
+If using custom JSON definition schema, users need to use Python dict to produce and consume messages, this is different from using Record definition 
+and the `_record_cls` param should be None when generating `AvroSchema` object.
+
+```
+schema_definition = load_schema("examples/company.avsc")
+# schema_definition = avro.schema.parse(open("examples/company.avsc", "rb").read()).to_json()
```
Comment (BewareMyPower) [inline site2/docs/client-libraries-python.md:263]: I just found this comment is confused. If you want to distinguish the `fastavro` and `avro`, you should better add the full imports. Like

```python
import fastavro

schema_definition = fastavro.schema.load_schema('examples/company.asvc')
```

or

```python
import avro

schema_definition = avro.schema.parse(open("examples/company.avsc", "rb").read()).to_json()
```

It's better to just retain only one in the example code. Don't use comment for the other one.

Comment (gaoran10) [reply site2/docs/client-libraries-python.md:263]: Good idea! Thanks.

**Review by merlimat — APPROVED** [review]

**Comment by Anonymitaet** [comment]
Talked with @gaoran10 just now, we are updating the doc, please do not merge this PR now.

**Review by codelipenghui — APPROVED** [review]

**Comment by Anonymitaet** [comment]
Hi @gaoran10 we've added docs in #12914, so I close this PR.

**Pull request closed without merging**